client/app/queue/OrganizationUsers.jsx

```diff
@@ -214,46 +215,39 @@ export default class OrganizationUsers extends React.PureComponent {
     });
   }
 
-  decisionDraftingButton = (user, attorney) =>
-    <span {...buttonPaddingStyle}><Button
-      name={attorney ? COPY.USER_MANAGEMENT_DISABLE_DECISION_DRAFTING_BUTTON_TEXT : COPY.USER_MANAGEMENT_ENABLE_DECISION_DRAFTING_BUTTON_TEXT}
-      id={attorney ? `Disable-decision-drafting-${user.id}` : `Enable-decision-drafting-${user.id}`}
-      classNames={attorney ? ['usa-button-secondary'] : ['usa-button-primary']}
-      loading={this.state.changingDecisionDrafting[user.id]}
-      onClick={this.modifyDecisionDrafting(user, !attorney)} /></span>
-
   adminButton = (user, admin) =>
-    <span {...buttonPaddingStyle}><Button
+    <div {...buttonStyle}><Button
       name={admin ? COPY.USER_MANAGEMENT_REMOVE_USER_ADMIN_RIGHTS_BUTTON_TEXT : COPY.USER_MANAGEMENT_GIVE_USER_ADMIN_RIGHTS_BUTTON_TEXT}
       id={admin ? `Remove-admin-rights-${user.id}` : `Add-team-admin-${user.id}`}
       classNames={admin ? ['usa-button-secondary'] : ['usa-button-primary']}
       loading={this.state.changingAdminRights[user.id]}
-      onClick={this.modifyAdminRights(user, !admin)} /></span>
+      onClick={this.modifyAdminRights(user, !admin)} /></div>
 
   removeUserButton = (user) =>
-    <span {...buttonPaddingStyle}><Button
+    <div {...buttonStyle}><Button
       name={COPY.USER_MANAGEMENT_REMOVE_USER_FROM_ORG_BUTTON_TEXT}
       id={`Remove-user-${user.id}`}
       classNames={['usa-button-secondary']}
       loading={this.state.removingUser[user.id]}
-      onClick={this.removeUser(user)} /></span>
+      onClick={this.removeUser(user)} /></div>
 
   mainContent = () => {
     const judgeTeam = this.state.judgeTeam;
     const listOfUsers = this.state.organizationUsers.map((user, i) => {
       const { judge, attorney, admin } = user.attributes;
-      const topStyle = i === 0 ? topUserStyle : {};
+      const style = i === 0 ? topUserStyle : userStyle;
 
       return <React.Fragment>
-        <li key={user.id} {...topStyle}>{this.formatName(user)}
+        <li key={user.id} {...style}>{this.formatName(user)}
           { judgeTeam && judge && <strong> ( {COPY.USER_MANAGEMENT_JUDGE_LABEL} )</strong> }
           { judgeTeam && attorney && <strong> ( {COPY.USER_MANAGEMENT_ATTORNEY_LABEL} )</strong> }
-          { judgeTeam && admin && <strong> ( {COPY.USER_MANAGEMENT_ADMIN_LABEL} )</strong> } &nbsp;</li>
-        <div {...buttonStyle}>
-          { judgeTeam && !judge && this.decisionDraftingButton(user, attorney) }
-          { this.adminButton(user, admin) }
-          { this.removeUserButton(user) }
-        </div>
+          { judgeTeam && admin && <strong> ( {COPY.USER_MANAGEMENT_ADMIN_LABEL} )</strong> }
+        </li>
+        { judgeTeam && admin ? <div {...topUserBorder}></div> :
+          <div {...buttonContainerStyle}>
+            { !judgeTeam && this.adminButton(user, admin) }
+            { this.removeUserButton(user) }
+          </div> }
```

Comment:
I have a hard time parsing this ternary operator - could we split the first part onto two lines and still be good JS style?

```suggestion
        { judgeTeam && admin ? 
          <div {...topUserBorder}></div> :
          <div {...buttonContainerStyle}>
            { !judgeTeam && this.adminButton(user, admin) }
            { this.removeUserButton(user) }
          </div> }
```

Reply:
Sure can!